tests/integration/Storefront/Controller/RobotsControllerTest.php

```diff
@@ -29,6 +29,8 @@ public function testRobotsTxt(): void
         $appUri = parse_url($appUrl)['path'] ?? '';
 
         static::assertIsString($html);
-        static::assertSame("User-agent: *\n\nAllow: /\n\nDisallow: /*?\n\nAllow: /*theme/\n\nAllow: /media/*?ts=\n\nDisallow: {$appUri}/account/\nDisallow: {$appUri}/checkout/\nDisallow: {$appUri}/widgets/\nAllow: {$appUri}/widgets/cms/\nAllow: {$appUri}/widgets/menu/offcanvas\n\nSitemap: {$appUrl}/sitemap.xml", $html);
+        // Default rules block followed by domain-specific rules block, separated by double newline
+        $expected = "User-agent: *\n\nAllow: /\nDisallow: /*?\nAllow: /*theme/\nAllow: /media/*?ts=\n\n\nDisallow: {$appUri}/account/\nDisallow: {$appUri}/checkout/\nDisallow: {$appUri}/widgets/\nAllow: {$appUri}/widgets/cms/\nAllow: {$appUri}/widgets/menu/offcanvas\n\nSitemap: {$appUrl}/sitemap.xml";
```

Comment:
can't be put the expected output into a [nowdoc](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.nowdoc) block for better readability?